packages/applications-service-api/src/services/document.service.js

```diff
@@ -47,11 +47,56 @@ const getDocuments = async (caseRef, pageNo, searchTerm) => {
   return documents;
 };
 
-const getOrderedDocuments = async (caseRef, pageNo) => {
+const getOrderedDocuments = async (caseRef, pageNo, searchTerm, stage, type) => {
   const { itemsPerPage: limit } = config;
   const offset = (pageNo - 1) * limit;
 
-  const where = { case_reference: caseRef };
+  let where = { case_reference: caseRef };
+  if (stage) where = { ...where, Stage: { [Op.in]: stage } };
+  if (type) where = { ...where, filter_1: { [Op.in]: type } };
```

Comment:
What's the difference between this where condition and that on line 90?

Reply:
On line 90 we use AND construct - for the orOptions AND the rest. Line 55 translates to `{ case_reference: 'TR040009', filter_1: { [Symbol(in)]: [ 'f1', 'f2' ] } } `, whilst line 84 translates to `{ [Symbol(and)]: [ { case_reference: 'TR040009' }, { filter_1: [Object] } ] }`